### ISSUE TITLE
use-mutative installed old version of mutative

### ISSUE THREAD

**Issue by alexjurkiewicz** [unadlib/use-mutative#6]
When I run `npm install mutative use-mutative`,  an old version of `mutative` is installed:
```
$ npm list | egrep '(mutative|react)'
├── @types/react-dom@18.2.18
├── @types/react@18.2.48
├── @vitejs/plugin-react-swc@3.5.0
├── eslint-plugin-react-hooks@4.6.0
├── eslint-plugin-react-refresh@0.4.5
├── mutative@0.2.2
├── react-dom@18.2.0
├── react-draggable@4.4.6
├── react-error-boundary@4.0.12
├── react-tooltip@5.26.0
├── react@18.2.0
├── use-mutative@0.0.4
```

Trying to force install the latest version of mutative doesn't work.

```
❯ npm instal mutative@1.0.3 use-mutative
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: project@0.0.0
npm ERR! Found: mutative@1.0.3
npm ERR! node_modules/mutative
npm ERR!   mutative@"1.0.3" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer mutative@"^0.2.2" from use-mutative@0.0.4
npm ERR! node_modules/use-mutative
npm ERR!   use-mutative@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by alexjurkiewicz** [comment]
Thank you, and congratulations!! 🎊

**Comment by unadlib** [comment]
hi @alexjurkiewicz , we have released `use-mutative` version 1.0. Feel free to use it!
